src/palace/manager/api/overdrive/api.py

```diff
@@ -697,18 +710,17 @@ def patron_request(
         data: str | None = None,
         exception_on_401: bool = False,
         method: str | None = None,
-        is_fulfillment: bool = False,
+        palace_context: bool = False,
```

Comment:
I updated this parameter because we use these credentials to do other things then just fulfillment. They are used to get the patrons loans during availability sync and do checkouts.